test/fixtures/simple_regional_private/example.tf

```diff
@@ -24,5 +24,4 @@ module "example" {
   subnetwork                     = "${google_compute_subnetwork.main.name}"
   ip_range_pods                  = "${google_compute_subnetwork.main.secondary_ip_range.0.range_name}"
   ip_range_services              = "${google_compute_subnetwork.main.secondary_ip_range.1.range_name}"
-  compute_engine_service_account = "${var.compute_engine_service_account}"
```

Comment:
Is this causing problems too?

Reply:
I'd like to get us into the habit of relying on the default behavior more often in fixtures. Default is an auto-generated service account.

Reply:
Wholly agreed! 👍